_faqs/2020-04-17-how-can-i-directly-open-an-image-file-in-greenshot.markdown

```diff
@@ -0,0 +1,11 @@
+---
+layout: faq
+status: publish
+published: true
+title: How can I open an image in Greenshot directly?
```

Comment:
```suggestion
title: How can I open an image in Greenshot on double click?
```

Reply:
imho 'on double click' is not correct, as it is only one option beside 'open with' that is clearly descriped later in the text: "From then, by right clicking on an image and select *Open with* brings Greenshot as an option to choose for open the file."

Reply:
I would almost call it "How can I open an image file in Greenshot"